### PR TITLE
roles: add `passkeyUser` objectClass only once

### PR DESCRIPTION
**Pull request by ikerexxe**
When calling `passkey_add()` append the new passkey mapping data.

### CODE REVIEW

**Comment by ikerexxe** [comment]
For reference I would like to avoid creating LDAP-specific code in test cases, as indicated at https://github.com/SSSD/sssd/pull/6965#discussion_r1426461106.

File: sssd_test_framework/roles/ldap.py [hunk 1]
```diff
@@ -624,6 +624,9 @@ def __init__(self, role: LDAP, name: str, basedn: LDAPObject | str | None = "ou=
         """
         super().__init__(role, name, f"cn={name}", basedn, default_ou="users")
 
+        self.passkey_object = False
```
Comment (justin-stephenson) [inline sssd_test_framework/roles/ldap.py:627]: IMO a name such as  `first_passkey_add` makes more sense. or `initial_passkey`

**Review by justin-stephenson — DISMISSED** [review]

File: sssd_test_framework/roles/ldap.py (hunk 1 above)
Comment (ikerexxe) [reply sssd_test_framework/roles/ldap.py]: Changed

**Review by justin-stephenson — APPROVED** [review]

**Comment by madhuriupadhye** [comment]
I tested the patch, looks good to me.

**Review by madhuriupadhye — APPROVED** [review]

**Pull request merged**